shared/components/GenericList/Pagination/Pagination.js

```diff
@@ -30,6 +43,8 @@ export const Pagination = ({
 }) => {
   const pagesCount = Math.ceil(itemsTotal / itemsPerPage);
 
+  const partitions = makePartitions(currentPage, pagesCount);
+
   return (
     <div style={{ display: 'flex', justifyContent: 'center' }}>
```

Comment:
It would be nice to have pagination in one line so maybe add `align-items: baseline;` here.

But also colors do not match:
- arrows should be blue
- items and active page are in different color
- there should be no underline

<img width="229" alt="Screen Shot 2021-11-26 at 12 17 27" src="https://user-images.githubusercontent.com/20790348/143574610-45f7a650-2cff-4af6-9913-159d1701bcd9.png">
<img width="498" alt="Screen Shot 2021-11-26 at 12 17 16" src="https://user-images.githubusercontent.com/20790348/143574619-fb0cd321-4c6c-403b-b00d-5df901519ec7.png">